frontend/src/components/MapView/LeftPanel/AnalysisPanel/ExposureAnalysisActions/index.tsx

```diff
@@ -81,6 +85,43 @@ function ExposureAnalysisActions({
     [analysisDefinition, exposureAnalysisCsvData],
   );
 
+  const handleDownloadReport = async () => {
```

Comment:
let's add a bit of context here

Reply:
I added some comment, don't hesitate if any suggestion or want some more details